source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/TabBaseViewModel.cs

```diff
@@ -1071,6 +1071,51 @@ internal void AddTextToMap(IGeometry geom, string text)
             RaisePropertyChanged(() => HasMapGraphics);
         }
 
+        internal void AddTextToMap(IGeometry geom, string text, double angle, AzimuthTypes azimuthType)
+        {
+            var mxDoc = ArcMap.Application.Document as IMxDocument;
+            var av = mxDoc.FocusMap as IActiveView;
+            var gc = av as IGraphicsContainer;
+            double bearing = 0.0;
+            if(azimuthType == AzimuthTypes.Mils)
```

Comment:
Is there a method that already does this calculation/conversion somewhere that could be used instead?